### PR TITLE
Upgrade to polymer 1.0

### PR DESCRIPTION
**Pull request by crhym3**
Also, rename `dontSendOnClose` to `manual` since the upgrade is breaking anyway.
Test with `wct`.

### CODE REVIEW

**Comment by crhym3** [comment]
Fixes #4

File: beacon-send.html [hunk 1]
```diff
@@ -40,89 +40,86 @@
...
+    properties: {
+
+      /**
+       * The url of the remote server to send data to.
+       *
+       * @attribute url
+       * @type string
+       * @default null
+       */
+      url: String,
```
Comment (ebidel) [inline beacon-send.html]: Let's give these default val:

    url: {
      type: String,
      value: null
    },

File: beacon-send.html [hunk 2]
```diff
@@ -40,89 +40,86 @@
...
+     * @param {Object} detail
+     *   @param {string} detail.success True if the message was queued successfully.
+     */
+
+    properties: {
+
+      /**
+       * The url of the remote server to send data to.
+       *
+       * @attribute url
```
Comment (ebidel) [inline beacon-send.html]: you can remove `@attribute` `@type` and `@default` from the doc strings. They are picked up from the actual code now.

File: beacon-send.html [hunk 3]
```diff
@@ -40,89 +40,86 @@
...
+      url: String,
+
+      /**
+       * The data to send to the remove server.
+       *
+       * @attribute data
+       * @type ArrayBufferView|Blob|String|FormData
+       * @default null
+       */
+      data: Object,
```
Comment (ebidel) [inline beacon-send.html]: same will all of these

File: beacon-send.html [hunk 4]
```diff
@@ -40,89 +40,86 @@
...
+
+    created: function() {
+      if (!('sendBeacon' in navigator)) {
+        console.warn(this.localName + ': the Beacon API is not available in this browser.');
+      }
+    },
+
+    ready: function() {
+      this._unloadHandler = this.send.bind(this);
+      this._manualChanged();
```
Comment (ebidel) [inline beacon-send.html]: you might not need this call anymore

File: bower.json [hunk 5]
```diff
@@ -24,7 +24,9 @@
     ".bowerrc"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^0.4.1",
-    "polymer-test-tools": "Polymer/polymer-test-tools#^0.4.1"
+    "polymer": "Polymer/polymer#1.0.3"
```
Comment (ebidel) [inline bower.json]: `#^1.0.3`

File: demo/index.html [hunk 6]
```diff
@@ -3,9 +3,12 @@
 <head>
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
   <title>beacon-send element demo</title>
-  <script src="../platform/platform.js"></script>
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
```
Comment (ebidel) [inline demo/index.html]: webcomponents-lite.min.js

File: demo/index.html [hunk 7]
```diff
@@ -3,9 +3,12 @@
...
+
   <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=RobotoDraft">
```
Comment (ebidel) [inline demo/index.html]: RobotoDraft -> Roboto

File: test/beacon-auto.html [hunk 8]
```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <title>beacon-send basic test</title>
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
```
Comment (ebidel) [inline test/beacon-auto.html]: webcomponents-lite.min.js

File: test/beacon-manual.html [hunk 9]
```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <title>beacon-send basic test</title>
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
```
Comment (ebidel) [inline test/beacon-manual.html]: webcomponents-lite.min.js

**Comment by ebidel** [comment]
Reviewed. Thanks for this!

File: beacon-send.html (hunk 4 above)
Comment (crhym3) [inline beacon-send.html]: interesting. if I remove it, the test with `window.dispatchEvent` fails. it looks like the observer isn't called on ready. is that expected:

screen shot 2015-06-17 at 12 27 29 am

File: beacon-send.html (hunk 1 above)
Comment (crhym3) [inline beacon-send.html]: done

File: beacon-send.html (hunk 2 above)
Comment (crhym3) [inline beacon-send.html]: done

File: beacon-send.html (hunk 3 above)
Comment (crhym3) [inline beacon-send.html]: done

File: bower.json (hunk 5 above)
Comment (crhym3) [inline bower.json]: done

File: demo/index.html (hunk 6 above)
Comment (crhym3) [inline demo/index.html]: done

File: demo/index.html (hunk 7 above)
Comment (crhym3) [inline demo/index.html]: done

File: test/beacon-auto.html (hunk 8 above)
Comment (crhym3) [inline test/beacon-auto.html]: done

**Comment by crhym3** [comment]
PTAL

**Comment by ebidel** [comment]
LGTM

**Pull request merged**